web_src/js/utils/dom.js

```diff
@@ -227,3 +227,15 @@ export function initSubmitEventPolyfill() {
   document.body.addEventListener('click', submitEventPolyfillListener);
   document.body.addEventListener('focus', submitEventPolyfillListener);
 }
+
+/**
+ * Check if an element is visible, equivalent to jQuery's `:visible` pseudo.
+ * Note: This function doesn't account for all possible visibility scenarios.
+ * @param {HTMLElement} element The element to check.
+ * @returns {boolean} True if the element is visible.
+ */
+export function isVisible(element) {
```

Comment:
Maybe it should be called "isElemVisible" to match "showElem" and "hideElem" (a nit)

Reply:
Done https://github.com/go-gitea/gitea/pull/29271/commits/f658668951a7934f5e362b0b0d9526b67ecf17b2